website/source/docs/vault-enterprise/ui/index.html.md

```diff
@@ -0,0 +1,78 @@
+---
+layout: "docs"
+page_title: "Vault Enterprise UI"
+sidebar_current: "docs-vault-enterprise-ui"
+description: |-
+  Vault Enterprise features a user interface for interacting with Vault. Easily
+  create, read, update, and delete secrets, authenticate, unseal, and more with
+  the Vault Enterprise UI.
+---
+
+# Vault UI
+
+Vault Enterprise features a user interface for interacting with Vault. Easily
+create, read, update, and delete secrets, authenticate, unseal, and more with
+the Vault Enterprise UI.
+
+To use the UI, you must have an active or trial Vault Enterprise or Vault Pro
+trial. To start a trial, contact [HashiCorp sales](mailto:sales@hashicorp.com).
```

Comment:
"you must have an active or trial Vault Enterprise or Vault Pro trial" -- was the last "trial" meant to be "license"? Or "an active Vault Enterprise or Vault Pro trial license"? Or something else?